modules/distribution-zones/src/test/java/org/apache/ignite/internal/distributionzones/DistributionZoneManagerConfigurationChangesTest.java

```diff
@@ -356,10 +211,15 @@ void testZoneDeleteDoNotRemoveMetaStorageKey() throws Exception {
         assertDataNodesForZone(1, nodes, keyValueStorage);
     }
 
-    private void mockVaultZonesLogicalTopologyKey(Set<String> nodes) {
-        byte[] newLogicalTopology = toBytes(nodes);
+    private static TablesConfiguration mockTablesConfiguration() {
```

Comment:
I'm pretty sure this mocking is not needed at all, we can replace it with `@InjectConfiguration`

Reply:
Looks like this comment hast not been fixed, why did you resolve it?